app/test/unit/find-forked-remotes-to-prune-test.ts

```diff
@@ -42,7 +42,7 @@ function createSampleBranch(name: string, upstream: string | null) {
     author,
   }
 
-  return new Branch(name, upstream, branchTip, BranchType.Local)
+  return new Branch(name, upstream, branchTip, BranchType.Local, '')
```

Comment:
Same: if you agree with my suggestion above:
```suggestion
  return new Branch(name, upstream, branchTip, BranchType.Local, null)
```

Reply:
Check the rest of the changes, because there would be more `null`s to add and `remoteName: '',` to remove 😅